kcidb/test_db.py

```diff
@@ -1,7 +1,5 @@
 """kcdib.db module tests"""
-
```

Comment:
Please do not include unnecessary whitespace changes into unrelated commits. If e.g. CI requires it, please make a separate commit for it.